floor_generator/lib/processor/view_processor.dart

```diff
@@ -43,10 +42,12 @@ class ViewProcessor extends QueryableProcessor<View> {
     final query = classElement
         .getAnnotation(annotations.DatabaseView)
         .getField(AnnotationField.VIEW_QUERY)
-        .toStringValue();
+        ?.toStringValue()
```

Comment:
Is this `?` really neccessary here? Isn't the `query` field mandatory for `DatabaseView` annotations?

Reply:
Yes, `query` is mandatory. Unluckily, the code generator can still be run even if the code doesn't compile because the `query` is missing. In that case, the gernator produces a helpful error message.